CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Prospector Changelog
 =======
 
+## Version 1.1.4
```

Comment:
I think that you forked from master butI forgot to merge master into develop with that release. That is bad of me, sorry! I was just trying to get a small fix out ASAP with it breaking builds for people and forgot that step.

Reply:
Probably worth a master->develop merge too , I can do it if you like or if you're on the case anyway no worries.

Reply:
Just merged :)

Reply:
Guess I'll have to rebase to remove these commits